manager/controllers/advisories.go

```diff
@@ -188,15 +188,15 @@ func AdvisoriesListIDsHandler(c *gin.Context) {
 func buildQueryAdvisories(db *gorm.DB, account int) *gorm.DB {
 	query := db.Table("advisory_metadata am").
 		Select(AdvisoriesSelect).
-		Joins("JOIN advisory_account_data aad ON am.id = aad.advisory_id and aad.systems_affected > 0").
+		Joins("JOIN advisory_account_data aad ON am.id = aad.advisory_id and aad.systems_installable > 0").
```

Comment:
if we need to show installable + applicable then we need something like 
```suggestion
		Joins("JOIN advisory_account_data aad ON am.id = aad.advisory_id and (aad.systems_installable > 0 or aad.systems_applicable > 0) ").
```